src/content/1.7/release/release-a-new-version-of-prestashop.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release a new version of PrestaShop
+weight: 4
+---
+
+# Release a new version of PrestaShop
+
+## Overview
+
+What does it mean, "to release a new version of "PrestaShop" ? Actually many different answers are eligible.
+
+You could say that "a new version of PrestaShop is released when there is a new release on the GitHub repository". Or you could say "a new version of PrestaShop is released when I can download a new version on prestashop.com". You could also say "a new version of PrestaShop is released when the auto-upgrade module allows to upgrade to this new version".
+
+So let's first put some words on all of these statements, to avoid any confusion.
+
+Steps to release a new version of PrestaShop:
+
+1. Create a new Build of PrestaShop, which is a ZIP archive which contains the software
```

Comment:
> the software + compiled assets (js, css) + third party dependencies (js, php)

This is my definition of "the software" 😄  but indeed what "software" exactly means can be confusing, since PrestaShop is used to refer to the company, the project, the repository, the source code and the ZIP archive. I will clear this.

Reply:
That's worth it to provide the details given by Pablo in the article